d3a_interface/constants_limits.py

```diff
@@ -220,13 +220,15 @@ class GlobalConfig:
     grid_fee_type = ConstSettings.IAASettings.GRID_FEE_TYPE
 
 
+class HeartBeat:
+    CHANNEL_NAME = "d3a-heartbeat"
+    RATE = 5  # in secs
```

Comment:
Didnt we decide in `INTERVAL` in the other PR? I personally prefer it.

Reply:
I can recall [this](https://github.com/gridsingularity/d3a-interface/pull/100#discussion_r440686661) where `HEARTBEAT_RATE ` was suggested

Reply:
OK, then ignore my comment.